packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -272,6 +294,14 @@ const WorkflowsView = defineComponent({
 				source: 'Workflows list',
 			});
 		},
+		openTemplateRepository(category: string) {
+			const url = this.templatesStore.getWebsiteCategoryURL(category);
+			this.$telemetry.track('User clicked Browse Sales Templates', {
+				role: this.usersStore.currentUserCloudInfo?.role,
+				active_workflow_count: this.workflowsStore.activeWorkflows.length,
+			});
+			window.open(url, '_blank');
```

Comment:
We should use plain `<a>` tag instead of navigating programmatically. That way all native browser interactions work as well (right click, cmd+click, etc.)

Reply:
Yeah, agree. Will update but the requirement is to open this in new tab so the link will have `target="_blank"`